features/vat-replacement/run.js

```diff
@@ -32,24 +29,22 @@ async function getTaxCategories (ctpClient) {
 
     try {
         let taxCategories = []
-        for await (const batch of ctpClient.fetchGraphQlBatches(params)) {
-            await Promise.map(batch, async (taxCategory) => {
-                taxCategories.push(taxCategory)
-            }, { concurrency: 4 })
```

Comment:
why don't we use concurrency here?

Reply:
At this point there is no async operation anymore so usage of promises is obsolete. In general one handles the concurrency with the client setting here: https://github.com/commercetools/commercetools-cli-scripts/pull/6/files#diff-a26699db6c2e410369f82f0a5872d3e883e977fcb542b962c1cd06ce7f84b427R31